dockerfiles/models/Dockerfile

```diff
@@ -34,14 +34,13 @@ ENV OMPI_ALLOW_RUN_AS_ROOT=1
 ENV OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
 
 # pytorch caused cuda download, this guarantees cpu version:
-RUN python -m pip install torch==1.7.0+cpu torchvision==0.8.1+cpu \
+RUN python -m pip install torch==1.7.1+cpu torchvision==0.8.2+cpu \
     -f https://download.pytorch.org/whl/torch_stable.html
 
-ARG TENSORFLOW_VERSION=2.4.1
-# Not using intel-tensorflow cause they have only 2.4.0 and we need 2.4.1
-RUN python -m pip install -U tensorflow~=${TENSORFLOW_VERSION} mxnet
+ARG TENSORFLOW_VERSION=2.7.0
+RUN python -m pip install -U intel-tensorflow~=${TENSORFLOW_VERSION} mxnet
```

Comment:
why intel-tensorflow?

Reply:
intel-tensorflow is a CPU optimization for tensorflow by intel and is also half the size of the regular tensorflow package (~500 mb vs ~250 mb)
It seems by the comment that we reluctantly move to `tensorflow` since `intel-tensorflow`  was missing an important update.
Added comment why we use intel-tensorflow